eyeshade/workers/wallet.js

```diff
@@ -211,18 +211,24 @@ exports.workers = {
       , fee              : ...
       , votes            : ...
       , hash             : '...'
+      , cohort           : '...'
       }
     }
  */
   'contribution-report':
     async (debug, runtime, payload) => {
+      const cohort = payload.cohort
       const paymentId = payload.paymentId
       const viewingId = payload.viewingId
       const contributions = runtime.database.get('contributions', debug)
       const wallets = runtime.database.get('wallets', debug)
       let state
 
-      payload.probi = bson.Decimal128.fromString(payload.probi.toString())
+      if (cohort && runtime.config.testingCohorts.includes(cohort)) {
+        payload.probi = bson.Decimal128.fromString('0')
+      } else {
+        payload.probi = bson.Decimal128.fromString(payload.probi.toString())
```

Comment:
i suspect that the better idiom is

        payload.probi = bson.Decimal128.fromString(cohort && runtime.conmfig.testingCohorts.includes(cohort) ? payload.probi.toString() : "0")